src/components/Header.scss

```diff
@@ -1,9 +1,53 @@
 .header {
+  width: 100vw;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--white-two);
   display: flex;
-  gap: 2rem;
-  align-items: center;
-  height: 7.2rem;
-  width: 90%;
-  margin: 0 auto;
-  font-size: 1.5rem;
+}
```

Comment:
- [ ] Hi @zeff96, great effort working on the styling of the bookstore project. But currently your design look slightly different from what we have in the original Zeplin design. Below are highlight of some things you need to work on

1. On the header section the `h1` text `Bookstore CMS` is broken into 2 lines which is not what we have in the original design, Kindly remove the `width: 100vw` in your `header` class or replace it with `%`
2. On the right hand side of the original design there is a user icon there but this is currently missing in your own design, kindly fix it
3. On the Form part the text `ADD NEW BOOK` is broken into two lines, kindly remove the width
4. On the `Current chapter` and `Chapter 17`, kindly make it be on a straight and not broken into two line so that it can look like what we have in the original design.

| Your Design 👇 | Zeplin Design 👇|
| ------------- | ------------- |
| ![ade](https://user-images.githubusercontent.com/78634841/230453274-d40c2d27-6500-4417-bdbe-2c735a1c0d28.png) | ![orig](https://user-images.githubusercontent.com/78634841/230453346-d0ca4b96-6d4a-4d3e-8515-1af4a571e126.png) ![orig2](https://user-images.githubusercontent.com/78634841/230453389-2927e4c9-0da5-47d4-a4f4-ebc660140636.png) |

Reply:
All the issue mentioned have been resolved, thanks for the highlights